### ISSUE TITLE
Issue with "LeafletToolbar" when using "leaflet.distordableimage"

### ISSUE THREAD

**Issue by miyujach** [publiclab/Leaflet.DistortableImage#273]
Hi everybody, 
I've got this issue when using "distordable-image"

```
Uncaught ReferenceError: LeafletToolbar is not defined
    at eval (leaflet.distortableimage.js?20e9:1042)
    at Object../node_modules/leaflet-distortableimage/dist/leaflet.distortableimage.js 
```

Here is my leaflet dependencies use:

```
import 'leaflet/dist/leaflet.css';
import 'leaflet.icon.glyph';
import 'leaflet-rotatedmarker';
import 'leaflet-distortableimage';
```

I'm using the plugin like it's explain on the github repo...

  ```
const img = L.distortableImageOverlay(
      imageUrl,
      {
        // 'corners' is the only required option for this class
        corners: [
          L.latLng(51.52,-0.10),
          L.latLng(51.52,-0.14),
          L.latLng(51.50,-0.10),
          L.latLng(51.50,-0.14)
        ],
      }
    ).addTo(map);
```
Is it a bug or I'm just using it badly ?

Thank you.

**Comment by sashadev-sky** [comment]
Hi @miyujach ! Is that the full error message? 

Did you include this line from the demo?

```
L.DomEvent.on(img._image, 'load', img.editing.enable, img.editing);
```

**Comment by KhanaaZero** [comment]
I have the same problem and I have added this, but there is no change.

**Comment by sashadev-sky** [comment]
I think @rexagod has answered a question like this before. Can you advise here??

**Comment by rexagod** [comment]
@miyujach You need to import [`leaflet-toolbar`](https://www.npmjs.com/package/leaflet-toolbar) as well.

**Comment by KhanaaZero** [comment]
I get still the same error, although I added the import:
`import "leaflet" ;
import "leaflet-toolbar" ;
import "leaflet-distortableimage" ;`

**Comment by rexagod** [comment]
@KhanaaZero Can you share the repo so I can clone it and have a look? Thanks!

**Comment by whoisstan** [comment]
I am having the same issue:
`leaflet.distortableimage.js:1042 Uncaught ReferenceError: LeafletToolbar is not defined`

which is here:
`var EditOverlayAction = LeafletToolbar.ToolbarAction.extend({`

This is how I load it:
`import L from 'leaflet';  //: 1.5.1
import { GeoSearchControl, GoogleProvider } from 'leaflet-geosearch'; //2.7.0
import 'leaflet-toolbar'; //0.4.0-alpha.2
import 'leaflet-distortableimage';` //0.4.2

I tried various ways to solve without luck. The error happens during the import already.

**Comment by sashadev-sky** [comment]
noting this same issue was reported on Jan 27 https://github.com/publiclab/Leaflet.DistortableImage/issues/123

**Comment by rexagod** [comment]
Okay I can think of a few cases why could be happening, but to narrow it down, can @whoisstan, @miyujach, or @KhanaaZero, tell me the exact sequence of steps they did, that landed them into this error. Also, I'd be really helpful if anyone could paste the *full* error log here. Thanks for cooperating with us!

**Comment by KhanaaZero** [comment]
I get this error directly after importing your component.

**Comment by sashadev-sky** [comment]
@rexagod I think I reproduced the issue. In our devDependencies we are locked in to `leaflet-toolbar` at 0.3.0 but since then the namespace has been updated to `L.Toolbar2`, so people pulling it in will get an error. I'm going to make a PR to update it right now, let me know your thoughts

**Comment by sashadev-sky** [comment]
We just released version `0.5.0` on npm. I believe that this has resolved the problem! And please let me know if it has not
